pkg/sql/logictest/testdata/logic_test/prepare

```diff
@@ -1565,7 +1565,6 @@ PREPARE pscs13 AS SELECT $1::bigint, 13
 query T
 SELECT name FROM pg_catalog.pg_prepared_statements ORDER BY name
 ----
-pscs08
```

Comment:
Unfortunately we may have to update these tests memo sizes change, and it's a bit tedious. I can't think of a good solution for this though...